docs/cookbook/production-deployment.md

```diff
@@ -0,0 +1,308 @@
+```
+echo $'domains:
+    -   id: 1
+        name: <YOUR_DOMAIN_NAME_HERE>
+        locale: en
+' > app/config/domains.yml
+```
+For each domain we need to create config with domain url.
+```
+echo $'domains_urls:
+    -   id: 1
+        url: https://<YOUR_DOMAIN_HERE>
+' >  app/config/domains_urls.yml
+```
+We remove `.git` folder so the built image doesn't have additional data.
+After the `project-base` is setup correctly, we launch the build of php-fpm container by docker build command.
+```
+docker build \
+    -f ./docker/php-fpm/Dockerfile-production \
+    -t production-php-fpm \
+    --compress \
+    --build-arg www_data_uid=1000 --build-arg www_data_gid=1000 \
+    --build-arg github_oauth_token=PERSONAL_ACCESS_TOKEN_FROM_GITHUB \
+    .
+```
+Replace the `PERSONAL_ACCESS_TOKEN_FROM_GITHUB` string by the token generated on [Github -> Settings -> Developer Settings -> Personal access tokens](https://github.com/settings/tokens/new?scopes=repo&description=Composer+API+token).
+There are also settings for user id and group id under who the ownership of files will be because php-fpm execution needs workers that are not allowed to runs under root user group.
+With `f` parameter we set path to Dockerfile that builds image.
+With `t` parameter we set the name of built image.
+
+If we are building the image on different server than production server, we can push built image into docker repository of production server via ssh.
+We use `-oStrictHostKeyChecking=no` argument to have ssh connection without the prompt that asks about adding target server record into `known_hosts` ssh configuration.
+We also want to establish connection to the server without prompting for password so we will use [key exchange method](http://sshkeychain.sourceforge.net/mirrors/SSH-with-Keys-HOWTO/SSH-with-Keys-HOWTO-4.html).
+```
+docker save production-php-fpm | gzip | ssh -oStrictHostKeyChecking=no -i <PRIVATE_KEY_PATH> root@<YOUR_DOMAIN_HERE> 'gunzip | docker load'
+```
+
+### Deployment On Production Server
+
+#### First Setup Deploy
+
+Now we need to copy [`docker-compose-prod-deploy.yml.dist`](../../docker/conf/docker-compose-prod-deploy.yml.dist) into folder on the production server as `docker-compose.yml`.
+After the image is in the repository of the production server we create docker containers and run phing target `build-new`,
+to build application for production with clean DB and base data.
+We use parameter `-p` to specify the name of the project and prefix for the volumes so these will be easily accessible.
+There are named volumes created under path `/var/lib/docker/volumes/` and one persisted folder `production-content` in `PATH_TO_DIRECTORY_WHERE_IS_DOCKER_COMPOSE_YML_STORED` for all uploaded images and generated files that should not be removed.  
+We create persited folder with correct owner and symlink to web folder for static files
+```
+cd PATH_TO_DIRECTORY_WHERE_IS_DOCKER_COMPOSE_YML_STORED #(e.g. /var/www/html)
+mkdir /var/www/production-content
```

Comment:
In previous command you were talking about `PATH_TO_DIRECTORY_WHERE_IS_DOCKER_COMPOSE_YML_STORED`  now you are just talking about `var/www/html` choose one of them and use it all over the place.

Reply:
/var/www/html is for example
/var/www/production-content is path to the static shared folder